### PR TITLE
Merge benefits into one page

### PR DESCRIPTION
**Pull request by tyom**

### CODE REVIEW

**Comment by tyom** [comment]
test this please

**Comment by milror00** [comment]
retest this please

File: cla_public/templates/macros/form.html [hunk 1]
```diff
@@ -34,6 +34,7 @@
   as_label=False,
   class_='',
   has_label=True,
+  has_info=True,
```
Comment (tyom) [inline cla_public/templates/macros/form.html]: Is this still used?

**Comment by tyom** [comment]
LGTM

**Pull request merged**